src/Movie/Movie.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { detailedMovieApi } from 'src/api/detailedMovieApi';
+import MovieDetailed from 'src/MovieDetailed/MovieDetailed';
+import { updateSingleMovieData } from 'src/MovieDetailed/MovieDetailedActions';
+
+class Movie extends Component {
+  componentDidMount() {
+    const { dispatchUpdateSingleMovieData, match } = this.props;
+    const id = match && match.params ? match.params.id : 0;
+    detailedMovieApi(id).then((response) => {
```

Comment:
you shouldn't make api calls in components with `saga`

the idea is next:
you define action type for saga to fetch `movie`, connect this action with handler `takeEvery` or smth like, you handler will load movie and dispatch action to be caught by redux reducer to update store, and in component you dispatch an action for saga
to summarize:
1. component did mount - dispatch action that will be processed by saga
2. saga handler will load what you need and dispatch action with payload to update store
it's as per https://github.com/redux-saga/redux-saga#usage-example